modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Hint.js

```diff
@@ -10,57 +10,44 @@
  */
 
 import ClayIcon from '@clayui/icon';
+import ClayPopover from '@clayui/popover';
 import className from 'classnames';
 import PropTypes from 'prop-types';
 import React, {useRef, useState} from 'react';
 
-import Popover from './Popover';
-
-export default function Hint({align, message, position, secondary, title}) {
+export default function Hint({message, position = 'top', secondary, title}) {
 	const iconRef = useRef();
-	const [showTooltip, setShowTooltip] = useState(false);
-
-	const handleMouseEnter = () => {
-		setShowTooltip(true);
-	};
-	const handleMouseLeave = () => {
-		setShowTooltip(false);
-	};
+	const [showPopover, setShowPopover] = useState(false);
 
 	const hintClasses = className('p-1', 'small', {
 		'text-secondary': secondary,
 	});
 
 	return (
-		<>
-			<span
-				className={hintClasses}
-				onMouseEnter={handleMouseEnter}
-				onMouseLeave={handleMouseLeave}
-				ref={iconRef}
-			>
-				<ClayIcon
-					className="mr-1"
-					small="true"
-					symbol="question-circle"
-				/>
-			</span>
-
-			{showTooltip && (
-				<Popover
-					align={align}
-					anchor={iconRef.current}
-					header={title}
-					position={position}
+		<ClayPopover
+			alignPosition={position}
+			header={title}
+			onShowChange={setShowPopover}
+			show={showPopover}
+			trigger={
+				<span
+					className={hintClasses}
```

Comment:
Why not add `className` right here instead of creating a variable and then passing it here, we managed to avoid declaring a new variable just for that.

Something like that:
```suggestion
					className={className('p-1', 'small', {
						'text-secondary': secondary,
					})}
```

Reply:
Right! I'll change it 👍